letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -163,7 +163,8 @@ def prepare(self):
 
         temp_install(self.mod_ssl_conf)
 
-    def deploy_cert(self, domain, cert_path, key_path, chain_path=None):
+    def deploy_cert(self, domain, cert_path, key_path,
+        chain_path=None, fullchain_path=None): # pylint: disable=unused-argument
```

Comment:
Indentation seems to be broken here, you should add 4 more spaces. I believe this was not caught due to 6604a0ff and its `indent-after-paren` change. Same elsewhere.